crypto/cmp/cmp_client.c

```diff
@@ -315,23 +315,23 @@ static int poll_for_response(OSSL_CMP_CTX *ctx, int sleep, int rid,
             if (check_after < 0 || (uint64_t)check_after
                 > (sleep ? ULONG_MAX / 1000 : INT_MAX)) {
                 ERR_raise(ERR_LIB_CMP, CMP_R_CHECKAFTER_OUT_OF_RANGE);
-                if (BIO_snprintf(str, OSSL_CMP_PKISI_BUFLEN, "value = %jd",
-                                 check_after) >= 0)
+                if (snprintf(str, OSSL_CMP_PKISI_BUFLEN, "value = %jd",
+			     check_after) >= 0)
```

Comment:
It's awesome that the -Wformat breakage on macos catches wrong format strings now. This needs an (intmax_t) cast or fiddling with the ugly PRId64 macro.